base.cfg

```diff
@@ -94,7 +94,6 @@ output = ${buildout:directory}/etc/${:_buildout_section_name_}
 
 
 [versions]
-setuptools =
```

Comment:
Why this line was removed?

Reply:
Un-pinning setuptools normally makes troubles. I removed it along the way of fixing buildout to get out of the constant restart-loop where it tries to upgrade setuptools and zc.buildout.
I think this should be out but it could also be that the real fix was in requirements.txt.

Reply:
The restart loop happens because you have conflicting versions of `setuptools` and `zc.buildout` in your venv and in buildout.

The solution is to properly pin them in the requirements them in your virtualenv and unpin them in the requirements and having separeted bin folders for buildout and virtualenv:

See https://gitlab.com/syslabcom/quaive/quaivesaas.buildout/-/blob/9d1fb7cba5a63cd4ac96ca2f59260e9832aed1c2/config/base.cfg#L277-279

Unless you find a proof of the need for this change, I would prefer not to merge it.